bin/migrate-files/migrate-files.py

```diff
@@ -33,75 +35,219 @@
 
-# Get total number of files
+starting_issue_document_id = 0
+# Determine job resume status
+if os.path.isfile('last-issue-file.pickle'):
+    with open('last-issue-file.pickle', 'rb') as file:
+        starting_issue_document_id = pickle.load(file)
+    print('Starting issues from:', starting_issue_document_id)
 
 cursor = conn.cursor()
 
-try:
-    cursor.execute ('SELECT COUNT(*) FROM OATS.OATS_DOCUMENTS WHERE dbms_lob.getLength(DOCUMENT_BLOB) > 0')
-except cx_Oracle.Error as e:
-    error, = e.args
-    print(error.message)
-
-count = cursor.fetchone()[0]
-print('Count =', count)
-
-# # Execute the SQL query to retrieve the BLOB data and key column
-cursor.execute(f"""
-SELECT DOCUMENT_ID, ALR_APPLICATION_ID, FILE_NAME, DOCUMENT_BLOB, DOCUMENT_CODE, DESCRIPTION, DOCUMENT_SOURCE_CODE, UPLOADED_DATE, WHEN_UPDATED, REVISION_COUNT, dbms_lob.getLength(DOCUMENT_BLOB) as LENGTH
-FROM OATS.OATS_DOCUMENTS
-WHERE dbms_lob.getLength(DOCUMENT_BLOB) > 0
-AND DOCUMENT_ID > {starting_document_id}
-ORDER BY DOCUMENT_ID ASC
-""")
-
-# Set the batch size
-BATCH_SIZE = 10
-
-# Track progress
-documents_processed = 0
-last_document_id = 0
-
-try:
-    with tqdm(total=count, unit="file", desc="Uploading files to S3") as pbar:
-        while True:
-            # Fetch the next batch of BLOB data
-            data = cursor.fetchmany(BATCH_SIZE)
-            if not data:
-                break
-            # Upload the batch to S3 with a progress bar
-            for document_id, application_id, filename, file, code, description, source, created, updated, revision, length in data:
-                tqdm.write(f"{application_id}/{document_id}_{filename}")
-
-                with tqdm(total=length, unit="B", unit_scale=True, desc=filename) as pbar2:
-                    s3.upload_fileobj(file, ecs_bucket, f"migrate/{application_id}/{document_id}_{filename}",
-                        Callback=lambda bytes_transferred: pbar2.update(bytes_transferred),)
-                pbar.update(1)
-                last_document_id = document_id
-                documents_processed += 1
-except Exception as e:
-    print("Something went wrong:",e)
-    print("Processed", documents_processed,  "files")
-    
-    # Set resume status in case of interuption
-    with open('last-file.pickle', 'wb') as file:
-        pickle.dump(last_document_id, file)
-    
-    cursor.close()
-    conn.close()
-    exit()
-
-# Display results
-print("Process complete: Successfully migrated", documents_processed, "files.")
+# Set batch size
+batch_size = 10
+
+if starting_issue_document_id > 0:
+    issue_docs(starting_issue_document_id,batch_size,cursor)
+elif starting_planning_document_id > 0:
+    planning_docs(starting_planning_document_id,batch_size,cursor)
+    issue_docs(0,batch_size,cursor)
+else:
+    print('Starting applications from:', starting_document_id)
+    application_docs(starting_document_id,batch_size,cursor)
+    planning_docs(0,batch_size,cursor)
+    issue_docs(0,batch_size,cursor)
```

Comment:
why do you need this if else?why not to use 
`
application_docs(starting_document_id,batch_size,cursor)
planning_docs(starting_planning_document_id,batch_size,cursor)
issue_docs(starting_issue_document_id,batch_size,cursor)
`